configure.ac

```diff
@@ -121,15 +121,38 @@ AC_ARG_WITH([rpc],
     [with_rpc="auto"]
 )
 AS_IF([test x"$with_rpc" != "xno"], [
-    PKG_CHECK_MODULES([TIRPC],[libtirpc], [
-        AC_DEFINE([HAVE_RPCENT_H], [1], [Have <rpc/rpcent.h>.])
-    ], [
-        AS_IF([test x"$with_rpc" = "xyes"], [
-            AC_MSG_ERROR([RPC support requested but not found])
+    rpc_test="libc libtirpc"
+    AS_CASE(["$with_rpc"],
+	[yes], [],
+	[auto], [rpc_test="$rpc_test no"],
+	[*], [rpc_test="`AS_ECHO(["$with_rpc"]) | tr , ' '`"])
```

Comment:
`m4_translit` would handle this better
https://www.gnu.org/software/m4/manual/html_node/Translit.html

Reply:
I don't get what you mean... it's a shell variable, how do I apply `m4_translit` to it's contents?

Reply:
sorry, you're right of course.  i was thinking not forcing a literal via `[...]` would delay the parsing, but obviously that doesn't make sense since m4 won't be used after configure is generated.